.github/workflows/app.yml

```diff
@@ -0,0 +1,48 @@
+name: app
+on:
+  - merge_group
+  - push
+  - pull_request
+
+# Only build for latest push/PR unless it's main or release/
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' && !startsWith( github.ref, 'refs/heads/release/' ) && !startsWith( github.ref, 'refs/heads/gh-readonly-queue/' ) }}
+
+defaults:
+  run:
+    shell: bash
+
+permissions: {}
+
+jobs:
+  lint-and-build:
+    strategy:
+      matrix:
+        debian_version:
+          - bookworm
+    runs-on: ubuntu-latest
+    container: debian:${{ matrix.debian_version }}
+    steps:
+      - name: Install base dependencies
+        run: |
+          apt-get update && apt-get install --yes git make
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22.x"
```

Comment:
No objection, but cross-referencing this specification with your request for a policy about it in <https://github.com/freedomofpress/securedrop-protocol/pull/74#discussion_r2112761392>.

Reply:
https://github.com/freedomofpress/securedrop-dev-docs/issues/245